src/main/java/in/projecteka/devservice/bridge/model/BridgeServiceRequest.java

```diff
@@ -0,0 +1,23 @@
+package in.projecteka.devservice.bridge.model;
+
+import lombok.Builder;
+import lombok.Value;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+@Value
+@Builder
+public class BridgeServiceRequest {
+    @NotBlank(message = "hospital id can't be blank")
+    String id;
+    @NotBlank(message = "hospital name can't be blank")
+    String name;
+    @NotNull(message = "hospital name alias can't be empty")
+    List<String> alias;
```

Comment:
Service can be either  hospital/lab. May be you can rename hospital with service in the messages

Reply:
done